02.calendar/cal.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+import minimist from "minimist";
+
+const argv = minimist(process.argv.slice(2));
+const today = new Date();
+let year = today.getFullYear();
+let month = today.getMonth();
+let optionedYear = argv["y"];
+let optionedMonth = argv["m"];
+
+const makeCalHeader = (year, month) => {
+  return `      ${month + 1}月 ${year}
+日 月 火 水 木 金 土`;
+};
+
+const makeCalBody = (year, month) => {
+  let content = "";
+  let saturdayCount = 0;
+  const firstDate = new Date(year, month, 1);
+  const lastDate = new Date(year, month + 1, 0);
+
+  for (
+    let date = new Date(year, month, 1);
+    date <= lastDate;
+    date.setDate(date.getDate() + 1)
+  ) {
+    let displayDate = date.getDate().toString();
+
+    if (displayDate.length === 1) {
+      displayDate = ` ${displayDate}`;
+    }
+
+    if (date.getDay() === 6) {
+      displayDate = `${displayDate} \n`;
```

Comment:
条件分岐で処理する必要があるのは `displayDate` の後ろの区切り文字だけなので、その処理のみを分岐として表現したほうがよい気がします。38 行目についても同様です。また、スペースが余分に含まれているように見えます。

```suggestion
      displayDate = `${displayDate}\n`;
```